core/include/scipp/core/dataset.h

```diff
@@ -370,6 +383,7 @@ class SCIPP_CORE_EXPORT Dataset {
 
   void setCoord(const Dim dim, Variable coord);
   void setLabels(const std::string &labelName, Variable labels);
+  void setMasks(const std::string &labelName, Variable masks);
```

Comment:
This should be called `setMask` (singular) to be consistent with the other setters.